public/js/use.js

```diff
@@ -288,7 +288,7 @@ function setupUse(params) {
           var image = new Image();
           image.src = reader.result;
           image.onload = function () {
-            var resizedImage = (data.files[0]['size'] > 2000000) ? resize(image, 800) : this.src;
+            var resizedImage = resize(image, 800);
```

Comment:
I know 😞. I'm gonna update your branch separating the exif and resizing so that we have the rotation working.